main.py

```diff
@@ -1,14 +1,18 @@
-import requests
-bestMovies =[]
-for i in range(1,26):
+from requests import Session
+
+sess = Session()
+
+best_movies = []
+for i in range(26):
     params = {
         "page" : i
     }
     #print(params)
-    response = requests.get("http://moviesapi.ir/api/v1/movies",params = params)
-    for i in range(0,10):
-        print(response.json()["data"][i]['title'])
-        bestMovies.append(response.json()["data"][i]['title'])
+    response = sess.get("http://moviesapi.ir/api/v1/movies", params = params)
+    data = response.json()
+    for item in data['data']:

```

Comment:
you don't know how many items gonna return by the result. 

ps:
It might explicitly said 10 result are enough per page, anyway.